blog/electron-22-0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Electron 22.0.0
+date: 2022-11-22T00:00:00.000Z
+authors:
+    - name: vertedinde
+      url: 'https://github.com/vertedinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+    - name: georgexu99
+      url: 'https://github.com/georgexu99'
+      image_url: 'https://github.com/georgexu99.png?size=96'
+slug: electron-22-0
+
+---
+
+Electron 22.0.0 has been released! It includes upgrades to Chromium `108`, V8 `10.8`, and Node.js `16.17.1`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 22.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on Twitter, or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `108`
+    * [New in Chrome 108](https://developer.chrome.com/blog/new-in-chrome-108/)
+    * [New in Chrome 107](https://developer.chrome.com/blog/new-in-chrome-107/)
+    * [New in DevTools 108](https://developer.chrome.com/blog/new-in-devtools-108/)
+    * [New in DevTools 107](https://developer.chrome.com/blog/new-in-devtools-107/)
+* Node.js `16.17.1`
+    * [Node 16.17.1 blog post](https://nodejs.org/en/blog/release/v16.17.1/)
+* V8 `10.8`
+
+### Highlighted Features
+
+* Added a new UtilityProcess API to launch a child process with node integration. [#36089](https://github.com/electron/electron/pull/36089)
```

Comment:
I think this deserves its own section rather than a bullet point, same with any new top-level Electron module.

Reply:
We should also link to the relevant documentation that we added for folks to get started rather than just the initial PR.